ProxyAppsSuite/tests/Laghos.yaml

```diff
@@ -0,0 +1,250 @@
+laghos:
+    summary: Unstructured high-order finite element spatial discretization and explicit high-order time-stepping.
+    subtitle: "{{dim_runs.id}}_{{dim_runs.problem}}"
+    variables:
+         dim_runs:
+           - { id: test1, problem: '0', dim_flag: "-dim", dim_val: '2', rp: '3', ok: '', ot: '', tf: '0.75', s: '', assembly: "-fa"}
+           - { id: test2, problem: '0', dim_flag: "-dim", dim_val: '3', rp: '1', ok: '', ot: '', tf: '0.75', s: '', assembly: "-fa"}
+           - { id: test3, problem: '1', dim_flag: "-dim", dim_val: '2', rp: '3', ok: '', ot: '', tf: '0.8', s: '', assembly: "-fa"}
+           - { id: test4, problem: '1', dim_flag: "-dim", dim_val: '3', rp: '2', ok: '', ot: '', tf: '0.6', s: '', assembly: "-fa"}
+           - { id: test5, problem: '2', dim_flag: "-dim", dim_val: '1', rp: '5', ok: '', ot: '', tf: '0.2', s: '', assembly: "-fa"}
+           - { id: test6, problem: '3', dim_flag: "-m", dim_val: 'data/rectangle01_quad.mesh', rp: '2', ok: '', ot: '', tf: '3.0', s: '', assembly: "-fa"}
+           - { id: test7, problem: '3', dim_flag: "-m", dim_val: 'data/box01_hex.mesh', rp: '1', ok: '', ot: '', tf: '3.0', s: '', assembly: "-fa"}
+           - { id: test8, problem: '4', dim_flag: "-m", dim_val: "data/square_gresho.mesh", rp: '3', ok: '3', ot: '2', tf: '0.62831853', s: '7', assembly: "-pa"}
+    slurm:
+        num_nodes: 2
+    permute_on: dim_runs
+    run:
+        timeout: 2000
+        modules: [ gcc, openmpi/2.1.2 ]
+        cmds:
+            - cd Laghos
+            - if [[ {{dim_runs.problem}} -eq 4 ]]; then
+            -    srun -n 8 ./laghos --problem "{{dim_runs.problem}}" "{{dim_runs.dim_flag}}" "{{dim_runs.dim_val}}" -rp "{{dim_runs.rp}}" -ok "{{dim_runs.ok}}" -ot "{{dim_runs.ot}}" -tf "{{dim_runs.tf}}" -s "{{dim_runs.s}}" "{{dim_runs.assembly}}" &> {{dim_runs.id}}.out 
+            - else 
+            -    srun -n 8 ./laghos --problem "{{dim_runs.problem}}" "{{dim_runs.dim_flag}}" "{{dim_runs.dim_val}}" -rp "{{dim_runs.rp}}" -tf "{{dim_runs.tf}}" "{{dim_runs.assembly}}" &> {{dim_runs.id}}.out
+            - fi
+            - cd ..
+            - mv Laghos/{{dim_runs.id}}.out {{dim_runs.id}}.out
+            - cat {{dim_runs.id}}.out
+    build:
+        modules: [gcc, openmpi/2.1.2]
+        source_path: laghos
+        env:
+            CC: mpicc
+        cmds:
+            #HYPRE / METIS
+            - echo "beginning hypre and metis build"
+            - cd Laghos/
+            - make setup
```

Comment:
You may want to force the build script to bail if any of your make commands fail.
`make setup || exit 1`, etc.

Since the laghos build seems to be dependent on the others in this case, though, you're probably fine.